xml_models/xml_models.py

```diff
@@ -222,8 +225,8 @@ def __new__(mcs, name, bases, attrs):
         return new_class
 
     def _get_xpath(cls, field_impl):
-        return property(fget=lambda cls: cls._parse_field(field_impl),
-                        fset=lambda cls, value: cls._set_value(field_impl, value))
+        return property(fget=lambda self: self._parse_field(field_impl),
```

Comment:
Why is this changed to `self`?  `cls` is convention in meta classes and wouldn't `self` be undefined as the first param is `cls`?